src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -592,6 +592,14 @@ private function addService($id, $definition)
             $return[] = sprintf("@throws InactiveScopeException when the '%s' service is requested while the '%s' scope is not active", $id, $scope);
         }
 
+        if ($definition->isDeprecated()) {
+            if ($return && 0 === strpos($return[count($return) - 1], '@return')) {
+                $return[] = '';
+            }
+
+            $return[] = '@deprecated';
```

Comment:
I reverted the change I made here, as after a second thought, I don't think it had any sense to add the same message of the trigger error (`@deprecated The service %s is deprecated. You should stop using it, as it will soon be removed`).

I think a message following a `@deprecated` should not be the same . it should state from which version it is deprecated (`vX.Y.Z`), eventually a replacement (using something else instead). Not a generic message, like the one thrown in the triggered error.

And I don't think adding yet another template would be that good though...